blackbox/docs/sql/administration/privileges.txt

```diff
@@ -0,0 +1,138 @@
+.. highlight:: psql
+.. _administration-privileges:
+
+=======================
+Cluster Wide Privileges
+=======================
+
+.. note::
+
+    Privileges are only part of the CrateDB `Enterprise Edition`_.
+    When the CrateDB Enterprise Edition is disabled, there will be
+    no user privilege checks, and every query will be executed
+    without the validation of permissions.
+
+
+``Permission Types``
+====================
+
+When a user is created, he doesn't have any privileges. The superuser can then
+grant privileges to this user.
+
+Please note that only the superuser can ``GRANT`` and ``REVOKE`` privileges.
+The superuser bypasses all privileges and can access all tables in the cluster.
+Therefor, It is also not possible to ``GRANT`` and ``REVOKE`` privileges to the
+superuser.
+
+User permissions are applied on the whole cluster. There is no option to ``GRANT``
+or ``REVOKE`` privileges on object level, such as table or schema.
+
+
+``DQL``
+.......
+
+Granting ``Data Query Language (DQL)`` permission to a user, indicates that this
+user is allowed to execute ``SELECT`` and ``SHOW`` queries, as well as using
+the available user defined functions.
+
+``DML``
+.......
+
+Granting ``Data Manipulation Language (DML)`` permission to a user, indicates
+that this user is allowed to execute ``INSERT``, ``UPDATE`` and ``DELETE``
+queries. Users having ``DML`` permissions will still require ``DQL`` permission.
```

Comment:
I guess this this because DML involves read operations as well? Even if it's something like a insert from values?

I guess somehow this makes sense as we don't add schema/ table level permissions